Fabric.Identity.API/Startup.cs

```diff
@@ -208,6 +209,11 @@ public void ConfigureServices(IServiceCollection services)
             app.UseCors(FabricIdentityConstants.FabricCorsPolicyName);
 
             app.UseIdentityServer();
+            if (_appConfig.AzureAuthenticationEnabled)
```

Comment:
Could we encapsulate this check in the UseAzureIdentityProvider function? We are passing in the config there anyway - might help to keep this already messy class a little cleaner.

Reply:
ha, yes.   I was thinking the startup was big to make it more straightforward to read :-)